src/HotChocolate/ApolloFederation/src/ApolloFederation/FederationVersion.cs

```diff
@@ -13,6 +13,8 @@ public enum FederationVersion
     Federation23 = 2_3,
     Federation24 = 2_4,
     Federation25 = 2_5,
-    // Federation26 = 2_6,
-    Latest = Federation25,
-}
\ No newline at end of file
+    Federation26 = 2_6,
+    Federation27 = 2_7,
+    // default to latest-1
+    Default = Federation26,
```

Comment:
I think we still should keep latest in addition to default ... 

Default => Recommended Version
Latest => Latest Version

What do you think?